common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -90,9 +90,10 @@ enum eGlobals {
     PHY_RATE_100KB_MAX                      = 8666,
     PHY_RATE_100KB_MIN                      = 72,
     PHY_RATE_100KB_INVALID                  = 0,
-    BSS_STEER_DISASSOC_TIMER_MS             = 200, // ~200ms
-    BSS_STEER_IMMINENT_VALID_INT            = 100,
-    BSS_STEER_VALID_INT                     = 50,
+    BEACON_TX_TIME_MS                       = 100,
+    BSS_STEER_DISASSOC_TIMER_MS             = 2000,
+    BSS_STEER_IMMINENT_VALID_INT            = (BSS_STEER_DISASSOC_TIMER_MS / BEACON_TX_TIME_MS),
```

Comment:
This makes the validity time equal to the disassociation timer, right? However, I'd expect the validity time to be a bit higher, no?

Reply:
IMO it should be even a bit less in order to give the client a chance to disconnect on its own (if  some reason the client decides to wait all the validity period before he responds).
personally i never seen a client reply to BSS after more than 1 sec.